app/main.py

```diff
@@ -1,2 +1,79 @@
+from typing import Any
+
+
 class Dictionary:
-    pass
+
+    LOAD_FACTOR = 2 / 3
+    INITIAL_CAPACITY = 8
+
+    def __init__(self) -> None:
+        self.capacity = self.INITIAL_CAPACITY
+        self.size = 0
+        self.table = [None] * self.capacity
+
+    @property
+    def size(self) -> int:
+        return self._size
```

Comment:
But you don`t have `_size` arg in `__init__` method

Reply:
I have a setter for this property and it is in __init__. Can you explain in more details, please.

Reply:
Is it unclear, as we must have all exist attribute in __init__, so pls, rename `size` to `_size` in 11 line